pkg/runner/client.go

```diff
@@ -180,24 +180,20 @@ func CreateClient(cfg *Config) (*pgclient.Client, error) {
 	return client, nil
 }
 
-func isTimescaleDBOSS(conn *pgx.Conn) (bool, error) {
-	var (
-		isTimescaleDB bool
-		isLicenseOSS  bool
-	)
-	err := conn.QueryRow(context.Background(), "SELECT _prom_catalog.is_timescaledb_installed()").Scan(&isTimescaleDB)
+func getTimescaleLicenseDetails(conn *pgx.Conn) (isTimescaleDB, isLicenseOSS bool, err error) {
```

Comment:
```suggestion
func getDatabaseDetails(conn *pgx.Conn) (isTimescaleDB, isLicenseOSS bool, err error) {
```

nit: the function in addition to returning license type returns database "type"